corehq/apps/userreports/tests/test_pillow.py

```diff
@@ -187,9 +293,18 @@ def test_process_doc_from_sql(self, datetime_mock):
 
         CaseAccessorSQL.hard_delete_cases(case.domain, [case.case_id])
 
-    @patch('corehq.apps.userreports.specs.datetime')
-    @override_settings(TESTS_SHOULD_USE_SQL_BACKEND=True)
+    @mock.patch('corehq.apps.userreports.specs.datetime')
+    def test_process_deleted_doc_from_sql_chunked(self, datetime_mock):
+        self.pillow = get_kafka_ucr_pillow(processor_chunk_size=100)
+        self._test_process_deleted_doc_from_sql(datetime_mock)
```

Comment:
It would be cleaner to pass in the pillow rather that update the reference on `self`. That would also prevent interference with other tests if one of these failed.